syspagen/syspagen.c

```diff
@@ -51,11 +61,14 @@ typedef struct {
 
 
 struct {
-	addr_t pkernel;
-	off_t offs;
-	size_t maxsz;
+	int type;
+	syspage32_t *syspage32;
+	syspage64_t *syspage64;
+
+	unsigned long pkernel;
+	unsigned long offs;
 
-	syspage_t *syspage;
+	size_t maxsz;
```

Comment:
`size_t` in context of syspage is somewhat risky - it might be 32 bit on some machine. I'm afraid `unsigned long long` here too

Reply:
Thx, done